pkg/build/controller/image_change_controller_test.go

```diff
@@ -240,7 +240,7 @@ func TestBuildConfigInstantiatorError(t *testing.T) {
 	if actual, expected := bcInstantiator.newBuild.Parameters.Strategy.DockerStrategy.From.Name, "registry.com/namespace/imagename:newImageID123"; actual != expected {
 		t.Errorf("Image substitutions not properly setup for new build. Expected %s, got %s |", expected, actual)
 	}
-	if bcUpdater.buildcfg != nil {
+	if bcUpdater.buildcfg != nil && bcUpdater.updateCount > 1 {
```

Comment:
@bparees I just realized I didn't answer what you were asking. There's no way for updateCount to be  1 without bldCfg being set to non-nil. Will remove the extraneous condition.